components/fxa-client/src/config.rs

```diff
@@ -19,6 +19,7 @@ struct ClientConfigurationResponse {
 #[derive(Clone, Debug, Serialize, Deserialize)]
 struct OpenIdConfigurationResponse {
     authorization_endpoint: String,
+    introspection_endpoint: String,
```

Comment:
Some very old self-hosted FxA servers might not report this value. Probably fine not to worry about supporting that, but we might think about whether that would show up cleanly in client logs in a way that would help us identify the source of the problem.

Reply:
I believe `serde_json` will generate a helpful message saying the specific field is missing, which we'll bubble up to consumers.